fud2/src/main.rs

```diff
@@ -1,23 +1,35 @@
-use fud2::build_driver;
 use fud_core::{cli, DriverBuilder};
 
 fn main() -> anyhow::Result<()> {
     let mut bld = DriverBuilder::new("fud2");
-    build_driver(&mut bld);
+    // build_driver(&mut bld);
 
     // In debug mode, get resources from the source directory.
     #[cfg(debug_assertions)]
-    bld.rsrc_dir(manifest_dir_macros::directory_path!("rsrc"));
+    {
+        bld.rsrc_dir(manifest_dir_macros::directory_path!("rsrc"));
+        bld.plugin_dir(manifest_dir_macros::directory_path!("plugins"));
```

Comment:
Further down the extremely silly terminology path: if they are built into the base functionality for fud2 and ship with the executable, are they really "plugins" at all, in the sense that the user plugs them in? Or are they merely "scripts"? (As in, maybe we should call the directory `scripts` and this function `script_dir`.)